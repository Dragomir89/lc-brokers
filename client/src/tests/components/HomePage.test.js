import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import '../setupTests';
import HomePage from '../../components/HomePage';

test('should render HomePage correctly', () => {
    const wrapper = shallow(<HomePage/>);
    expect(toJSON(wrapper)).toMatchSnapshot()
});