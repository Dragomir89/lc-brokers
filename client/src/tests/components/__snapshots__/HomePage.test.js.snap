// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render HomePage correctly 1`] = `
<h1>
  Home Page
</h1>
`;
